s3/pkg/datastore/aws/aws.go

```diff
@@ -42,6 +42,10 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	DefaultRegion = "us-east-1"
```

Comment:
can this be taken from a conf file.. / incase user wants to change at runtime

Reply:
ok. suggestion noted. But this is for default case.